runner.py

```diff
@@ -670,10 +688,15 @@ def get_main_parser():
                 job_script_prologue=job_script_prologue,
             )
         elif "condor" in args.executor:
+            portopts = {}
+            if "brux" in args.executor:
+                import socket
+                portopts = {"port": 8786, "host": socket.gethostname()} 
```

Comment:
Indeed I think we can move the config for lxplus out (after [L638](https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/58/files#diff-34cde5e49b54d9b67952c80d3580b6c421ebcf16097996e215aef54e44f78562R638)

https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/58/files#diff-34cde5e49b54d9b67952c80d3580b6c421ebcf16097996e215aef54e44f78562R650-R655

which is also used here